src/components/library-app-2/book-form.js

```diff
@@ -1,218 +1,256 @@
-//                   <Text style={styles.label}>
-//                     {label} <Text style={styles.req}>*</Text>{' '}
-//                   </Text>
-//                   <TextInput
-//                     keyboardType={keyboardType}
-//                     placeholder={label}
-//                     style={styles.input}
-//                     value={value}
-//                     onChangeText={value => handleFieldChange(key, value)}
-//                   />
-//                 </View>
-//               ),
-//             )}
-//           </View>
-//         </ScrollView>
-//         <Pressable
-//           onPress={handleSubmit}
-//           style={({pressed}) => [styles.submitBtn]}>
-//           <Text style={styles.submitBtnText}>Submit</Text>
-//         </Pressable>
-//       </View>
-//     </KeyboardAvoidingView>
-//   );
-// }
+  const handleFieldChange = (field, value) => {
+    setFormValues(prevFormValues => ({...prevFormValues, [field]: value}));
+  };
 
-// const styles = StyleSheet.create({
-//   container: {
-//     flex: 1,
-//     marginTop: StatusBar.currentHeight || 40,
-//   },
-//   header: {
-//     backgroundColor: '#5DB3FF',
-//     height: 50,
-//     justifyContent: 'center',
-//     alignItems: 'center',
-//   },
-//   headerText: {
-//     textTransform: 'capitalize',
-//     color: '#800080',
-//     fontWeight: '800',
-//     fontSize: 25,
-//   },
-//   submitBtn: {
-//     height: 50,
-//     backgroundColor: '#90EE90',
-//     display: 'flex',
-//     justifyContent: 'center',
-//     alignItems: 'center',
-//   },
-//   submitBtnText: {
-//     textTransform: 'capitalize',
-//     color: '#000',
-//     fontWeight: '700',
-//     fontSize: 20,
-//   },
-//   form: {
-//     flex: 1,
-//     justifyContent: 'flex-start',
-//     alignItems: 'center',
-//   },
-//   formGroup: {
-//     width: '90%',
-//     marginVertical: 10,
-//   },
-//   checkboxContainer: {
-//     width: '90%',
-//     marginVertical: 10,
-//     flexDirection: 'row',
-//     alignItems: 'center',
-//   },
-//   checkbox: {
-//     marginRight: 10,
-//   },
-//   req: {
-//     color: 'red',
-//   },
-//   label: {
-//     fontWeight: '700',
-//     marginVertical: 10,
-//   },
-//   input: {
-//     padding: 10,
-//     borderColor: '#000',
-//     fontSize: 15,
-//     borderWidth: 2,
-//     height: 40,
-//   },
-//   dropdownBtnStyle: {
-//     width: '100%',
-//     height: 40,
-//     padding: 10,
-//     backgroundColor: '#FFF',
-//     borderWidth: 2,
-//     borderColor: '#000',
-//   },
-//   dropdownBtnTxtStyle: {color: '#444', textAlign: 'left', fontSize: 15},
-//   dropdownRowStyle: {backgroundColor: '#EFEFEF', borderBottomColor: '#C5C5C5'},
-//   dropdownRowTxtStyle: {color: '#444', textAlign: 'left'},
-// });
+  const validateForm = () => {
+    for (const field of reqFields) {
+      if (formValues[field].toString().trim() === '') {
+        Alert.alert('Enter required fields');
```

Comment:
Lets not handle via alert. Lets handle it like a web app. Show an error message below the field

Reply:
done